RecoCTPPS/TotemRPLocal/python/totemRPLocalReconstruction_cff.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+
+# geometry
+from Geometry.VeryForwardGeometry.geometryRP_cfi import *
+XMLIdealGeometryESSource.geomXMLFiles.append("Geometry/VeryForwardData/data/RP_Garage/RP_Dist_Beam_Cent.xml")
```

Comment:
The EventSetup does not use module labels when looking up data so I do not understand why your change would work.